packages/bootstrap-vue-next/src/components/BTable/BTableLite.vue

```diff
@@ -199,7 +199,11 @@
         </BTh>
       </BTr>
     </BTfoot>
-    <BTfoot v-else-if="$slots['custom-foot']">
+    <BTfoot
+      v-else-if="$slots['custom-foot']"
+      :variant="props.footVariant"
+      :class="props.tfootClass"
```

Comment:
```ts
const footerProps = computed(() => ({
  variant: props.footerVariant,
  class: props.tfootClass
})
```

Then

```suggestion
      v-bind="footerProps"
```

Apply this to both uses of the BTfoot component, which should make it more resilient to additions

Reply:
Done!
Thank you for your review :)